src/app/component/comments/components/add-comment/add-comment.component.html

```diff
@@ -1,28 +1,42 @@
-<form class="main-wrapper"
-      [formGroup]="addCommentForm"
-      [ngClass]="!commentId ? 'wrapper-comment': 'wrapper-reply'"
-      (ngSubmit)="onSubmit()">
+<form
+  class="main-wrapper"
+  [formGroup]="addCommentForm"
+  [ngClass]="!commentId ? 'wrapper-comment' : 'wrapper-reply'"
+  (ngSubmit)="onSubmit()"
+>
   <app-user-profile-image
     [imgPath]="avatarImage"
     [firstName]="firstName"
-    [additionalImgClass]="'comments'">
+    [additionalImgClass]="'comments'"
+  >
   </app-user-profile-image>
   <div class="input-submit" *ngIf="!commentId; else reply">
-    <textarea placeholder="{{ 'homepage.eco-news.comment.placeholder.add-a-comment' | translate }}"
-              formControlName="content"></textarea>
-    <button class="primary-global-button"
-            [disabled]="!addCommentForm.valid">
-      {{ "homepage.eco-news.comment.comment"| translate }}
+    <textarea
+      placeholder="{{
+        'homepage.eco-news.comment.placeholder.add-a-comment' | translate
+      }}"
+      formControlName="content"
+    ></textarea>
+    <button
+      class="primary-global-button"
+      style="width: 505px"
+      [disabled]="!addCommentForm.valid"
+    >
+      {{ "homepage.eco-news.comment.comment" | translate }}
     </button>
```

Comment:
I think property 'width' must attach to 'textarea', but not to 'button'.

bug story https://github.com/ita-social-projects/GreenCity/issues/2026

![image](https://user-images.githubusercontent.com/24696596/109539523-13da7b00-7aca-11eb-9387-7b06ad3775a9.png)

Reply:
✅ Typo has been fixed